ansible/group_vars/netlogon.yml

```diff
@@ -0,0 +1,8 @@
+sshd_AuthorizedKeysCommand: /usr/bin/netkeys --ID %u
+sshd_AllowGroupsAlways:
+  - wheel
+  - dante
```

Comment:
`dante`: What's the reason for the group? iiuc it has the same permissions as wheel.

Reply:
wheel is completely local, dante has equivalent permissions to wheel but is provided by the network.  I've learned from experience its a bad idea to mix and match remote users and local groups, or local users and remote groups.  That's what the duplication is here.

For this to be applied to other hosts there would be more groups allowed since dante is supposed to be *very* limited membership.